diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -186,7 +191,7 @@ open class DiktatJavaExecTaskBase @Inject constructor(
         // githubActions should have higher priority than a custom input
         if (diktatExtension.githubActions) {
             // need to set user.home specially for ktlint, so it will be able to put a relative path URI in SARIF
-            System.setProperty("user.home", project.projectDir.toString())
+            systemProperty("user.home", project.projectDir.toString())
```

Comment:
in maven plugin this works fine?

Reply:
Yes, this change doesn't affect maven plugin